alt_build.ps1

```diff
@@ -1,4 +1,4 @@
-Param([string]$Deploy)
+Param([string]$Deploy, [bool]$nozip)
```

Comment:
Wouldn't that be:

```pwsh
[Switch] $NoZip
```

And then you can just use `build.ps1` 😉

Reply:
Aahh even simpler this way. Thanks !